content/efcore/getting-started/samples/loading-related-data-tutorial/include-method/Program.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var book = context.Books
+                .Single(b => b.BookId == 1)
+                .Include(b => b.Authors);
+            Console.WriteLine(String.Format("{0} - {1}", book.Title, book.Author));
```

Comment:
To print out the actual information, need to add another `{2}` in the formatted string and change it from `book.Author` to `book.Author.FirstName, book.Author.LastName`